engine/src/test/java/io/zeebe/engine/processor/workflow/activity/ServiceTaskTest.java

```diff
@@ -47,6 +47,33 @@ private static BpmnModelInstance workflow(final Consumer<ServiceTaskBuilder> con
     return builder.endEvent().done();
   }
 
+  @Test
+  public void shouldCreateJobFromServiceTaskWithJobTypeExpression() {
+    // given
+    ENGINE
+        .deployment()
+        .withXmlResource(workflow(t -> t.zeebeTaskType("=\"test\"").zeebeTaskRetries(5)))
```

Comment:
Let's hide the expression prefix `=` behind the model API, as we did for the other attributes. I suggest having two methods `zeebeJobTypeExpression(expression)` and `zeebeJobType(staticValue)` - one for an expression and one for a static value.

Additionally, we could rename `zeebeTaskType()` to `zeebeJobType()` because it is called "job type" everywhere else.

```suggestion
        .withXmlResource(workflow(t -> t.zeebeJobTypeExpression("\"test\"")))
```

Reply:
will do